recipes/group_humann2_uniref_abundances_to_go/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: group_humann2_uniref_abundances_to_go
-  version: '1.2.3'
+  version: '1.3.0'
 
 source:
-  md5: 97d9b2fe8afb953c8e284e458e355eec
-  url: https://github.com/jraysajulga/group_humann2_uniref_abundances_to_GO/archive/v1.2.3.zip
+  md5: 3871d8900c0bb729073b84da13b1c2d2
```

Comment:
@bebatut I get an md5 code of a78d947cfff8e76297840dd6ce8bfe7f

Reply:
Nice catch! Thanks

Reply:
Hmmm... It's still failing for some reason?

Reply:
I have submitted my own [PR](https://github.com/bioconda/bioconda-recipes/pull/15162) with the same changes and the tests ran successfully for some reason. Might be because of your system. Are you on linux? @bebatut